pkg/image/validation/image_test.go

```diff
@@ -0,0 +1,102 @@
+package validation
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+)
+
+func TestValidateImage(t *testing.T) {
+	imageConfigDir, err := os.MkdirTemp("", "eib-image-tests-")
+	require.NoError(t, err)
+	defer func() {
+		_ = os.RemoveAll(imageConfigDir)
+	}()
+
+	testImagesDir := filepath.Join(imageConfigDir, "images")
+	err = os.Mkdir(testImagesDir, os.ModePerm)
+	require.NoError(t, err)
+
+	testBaseImageFilename := filepath.Join(testImagesDir, "base-image.iso")
+	_, err = os.Create(testBaseImageFilename)
+	require.NoError(t, err)
+
+	tests := map[string]struct {
+		ImageDefinition        image.Definition
+		ExpectedFailedMessages []string
+	}{
+		`complete valid definition`: {
```

Comment:
Nit: Using `` creates string literals (i.e. constants) and not string variables (i.e. ""). Is there a reason to prefer the former over the latter?

Reply:
Nah, I don't have a preference. I literally just saw this approach and wanted to try it out. But I can also see the argument for code consistency across the project, so if we want to standardize on the existing model of a name field in the struct, I'm completely fine with that.